requirements/docs.in

```diff
@@ -15,6 +15,7 @@ git+https://github.com/readthedocs/sphinx-hoverxref
 # Docs
 sphinxemoji==0.2.0
 sphinxcontrib-httpdomain==1.8.0
+sphinxcontrib-mermaid>=0.7.1,<0.8
```

Comment:
This file is not following the proper pattern we have in the rest of the dependencies file. We should not pin the dependencies in the `.in` files unless there is a particular restriction.

You could just use `sphinxcontrib-mermaid` here.

In another PR, we should remove all the `==.*` from this file and re run `pip-compile -U`

Reply:
dependency will be removed, see other comment :+1: